doc/api/fs.md

```diff
@@ -2503,7 +2503,7 @@ changes:
 
 Read data from the file specified by `fd`.
 
-`buffer` is the buffer that the data will be written to.
+`buffer` is the buffer that the data which was read from the fd will be written to.
```

Comment:
This line exceeds the maximum allowed line length.

```suggestion
`buffer` is the buffer that the data which was read from the fd will be written
to.
```

Reply:
i tried reduce the line length